src/controllers/audience/audienceController.ts

```diff
@@ -175,6 +184,20 @@ export class AudienceController {
 		}
 	}
 
+	deleteOrganisation() {
+		return async (req: Request, res: Response) => {
+			JsonpathService.jsonpath().value(
```

Comment:
BUG - Where is this service?
MAINTENANCE - we want to wrap the implementation behind an interface, not expose the package directly from the service.
IE : `jsonpathservice.translate(..,...)` => implementation would be `this.jsonpath().value(..,..)`